### DISCUSSION TITLE
Welcome to sverchok Discussions!

### DISCUSSION THREAD

**Discussion by nortikin** [nortikin/sverchok#4457] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by enzyme69** [comment]
Year is 2022, I am a YouTuber Blender Sushi, surviving the pandemic madness, hopefully can continue making 3D procedural nodes art using SVerchok and Geometry Nodes, and all kind of nodes in Blender. 

Ideally I am also able to use FreeCAD nodes in Mac, which I have not manage to install.

I want to create beautiful engaging abstract art that's unlike the boring realistic gentrified 3D world right now. I want to publish unusual Art in AR and VR.

Thank you for SVerchok and helping me with fear of coding :)

**Reply by nortikin** [reply]
is it still pandemia in Australia?

> Year is 2022, I am a YouTuber Blender Sushi, surviving the pandemic madness, hopefully can continue making 3D procedural nodes art using SVerchok and Geometry Nodes, and all kind of nodes in Blender.
> 
> Ideally I am also able to use FreeCAD nodes in Mac, which I have not manage to install.
> 
> I want to create beautiful engaging abstract art that's unlike the boring realistic gentrified 3D world right now. I want to publish unusual Art in AR and VR.
> 
> Thank you for SVerchok and helping me with fear of coding :)

**Comment by GeorgiKovachev** [comment]
Hello, 
Recently I found out about Sverchok, I am also a fairly new Blender user and found some tutorials on youtube. Gorgeous addon, not sure if here is the place to ask but if it's not please tell me where I can do that.  I am following this tutorial https://www.youtube.com/watch?v=0BJQEUUcdLQ&t=248s but as you can see on the image attached  I get an error that does not occur on the video
![Untitled-1](https://user-images.githubusercontent.com/17068425/178672787-9ff631f2-40b5-4d3f-b069-68febd66e80a.jpg). Can you help out or guide me where I can ask. Thank you in advance

**Reply by Durman** [reply]
Hello.
http://nortikin.github.io/sverchok/docs/introduction/sverchok_community.html#questions

Or you can open separate question discussion.

**Reply by zeffii** [reply]
@GeorgiKovachev  connect the polygon output-socket from the get_object_data node, and not the edges-socket  , without the information in the polygon socket, the object can't be sliced.
